From d4193f43c9d5ca09c3deeb4ef05fffc065da21bd Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Thu, 15 Jun 2017 14:44:01 +0800
Subject: [PATCH] xmlsec1: add new recipe

This enables the building of the examples directory
and it's installed as ptest.

Upstream-Status: Inappropriate [ OE ptest specific ]

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>

---
 examples/Makefile | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/examples/Makefile b/examples/Makefile
index 02480c2..62f190e 100644
--- a/examples/Makefile
+++ b/examples/Makefile
@@ -8,9 +8,19 @@ PROGRAMS = \
 	decrypt1 decrypt2 decrypt3 \
 	xmldsigverify
 
+ifndef CC
 CC	= gcc
 CFLAGS	+= -g $(shell xmlsec1-config --cflags) -DUNIX_SOCKETS -Wall -Wextra
 LDLIBS	+= -g $(shell xmlsec1-config --libs)
+endif
+
+CFLAGS  += -I../include -g $(shell PKG_CONFIG_PATH=.. pkg-config --cflags xmlsec1 ) -DUNIX_SOCKETS
+LDLIBS += -L../src/.libs -g $(shell PKG_CONFIG_PATH=.. pkg-config --libs xmlsec1 )
+
+DESTDIR = /usr/share/xmlsec1
+install-ptest:
+	if [ ! -d $(DESTDIR) ]; then mkdir -p $(DESTDIR); fi
+	cp * $(DESTDIR)
 
 all: $(PROGRAMS)
 
